core/src/time.rs

```diff
@@ -18,68 +18,26 @@
 //! Time related utils.
 
 use crate::Error;
-use chrono::format::Fixed;
-use chrono::format::Item;
-use chrono::format::Numeric;
-use chrono::format::Pad;
-use chrono::SecondsFormat;
-use chrono::Utc;
+use std::str::FromStr;
 
-/// DateTime is the alias for chrono::DateTime<Utc>.
-pub type DateTime = chrono::DateTime<Utc>;
+/// DateTime is the alias for `jiff::Timestamp`.
+pub type DateTime = jiff::Timestamp;
```

Comment:
If you're doing a breaking change anyway, maybe it makes sense to rename this to a `Timestamp`. I think `DateTime` is somewhat odd for just a simple instant in time. (It's definitely not the worst thing though.)

Reply:
Yes. Per https://github.com/apache/opendal-reqsign/pull/634#discussion_r2370741703 I'll do the rename.